otterai/indexer.py

```diff
@@ -74,6 +89,14 @@ def analyze_project_structure(index: Dict[str, List[str]], repo_root: str) -> st
         3. Coding standards and patterns observed
         4. Important dependencies and configurations
         5. Testing approach
+         
+         steps:
+         1. read the codebase structure
+         2. read the key files content
+         3. generate the analysis
+         4. Strictly check which language is used in the codebase do not any other language if not specified in the codebase.
+
+         I'll give you billion dollars if you do follow this instruction.
```

Comment:
💼 The instruction 'I'll give you billion dollars if you do follow this instruction' seems unprofessional. Consider rephrasing it to maintain a professional tone.